pkg/server/middleware/webhook.go

```diff
@@ -42,7 +44,6 @@ func Webhook(webhookService svcframework.Service, noun webhook.Noun, verb webhoo
 		}
 
 		// publish the webhook
-		whService := webhookService.(*webhook.Service)
-		go whService.PublishWebhook(noun, verb, buf)
+		go webhookService.PublishWebhook(c, noun, verb, buf)
```

Comment:
be careful passing the context here, I purposly removed it because on doing an integration test of webhooks, the context closes in the middle of this go routine, so it kills the POST

Reply:
we need it now to propagate any errors that are encountered, it's the gin context so may be different behavior. is there a way I can test the cancelling behavior?

Reply:
I agree that you shouldn't be passing it. The gin context shouldn't apply to the webhook publishing.